noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -433,15 +442,16 @@ pub global BASE_OR_MERGE_PUBLIC_INPUTS_LENGTH: u32 = CONSTANT_ROLLUP_DATA_LENGTH
     + PARTIAL_STATE_REFERENCE_LENGTH
     + 2 * SPONGE_BLOB_LENGTH
     + 5;
-pub global BLOCK_ROOT_OR_BLOCK_MERGE_PUBLIC_INPUTS_LENGTH: u32 = 2
-    * APPEND_ONLY_TREE_SNAPSHOT_LENGTH
+pub global BLOCK_ROOT_OR_BLOCK_MERGE_PUBLIC_INPUTS_LENGTH: u32 = APPEND_ONLY_TREE_SNAPSHOT_LENGTH
+    + APPEND_ONLY_TREE_SNAPSHOT_LENGTH
```

Comment:
This is an unrelated fix - `constants.in.ts` incorrectly parsed `2 \n * APPEND_ONLY_TREE_SNAPSHOT_LENGTH` as `2 + ...` (`+` instead of `*`), discovered when editing `BLOCK_ROOT_OR_BLOCK_MERGE_PUBLIC_INPUTS_LENGTH`